src/main/java/org/jenkinsci/plugins/mesos/listener/MesosRunListener.java

```diff
@@ -86,13 +88,14 @@ public void onCompleted(Run run, TaskListener listener) {
         String monitoringUrl = mesosSlave.getMonitoringURL();
         PrintStream logger = listener.getLogger();
 
-        logger.println();
         if(monitoringUrl != null) {
-          logger.println("Slave resource usage: " + monitoringUrl);
+          logger.println("\nSlave resource usage: " + monitoringUrl + "\n");
         } else {
-          logger.println("Slave resource usage is not available for this build.");
+          MesosCloud mesosCloud = mesosSlave.getCloud();
+          if(mesosCloud != null && !StringUtils.isBlank(mesosCloud.getGrafanaDashboardURL())){
+            logger.println("\nSlave resource usage is not available for this build.\n");
```

Comment:
This if clause prevents the warning msg "Slave resource usage is not available for this build" when no Grafana Dashboard Url is provided in the Jenkins config.

If the clause is not present, on Jenkins Instances with no Grafana Dashboard Url configured, this Message will appear in every build log.